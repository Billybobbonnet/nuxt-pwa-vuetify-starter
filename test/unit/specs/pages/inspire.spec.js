import { createLocalVue, shallowMount } from "@vue/test-utils";
import Vuetify from "vuetify";

import Inspire from "~/pages/inspire";

describe("Inspire", () => {
  let wrapper;

  beforeAll(() => {
    const localVue = createLocalVue();
    localVue.use(Vuetify);

    wrapper = shallowMount(Inspire, {
      localVue,
      stubs: ["nuxt-logo", "vuetify-logo"]
    });
  });

  it("renders to a snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
