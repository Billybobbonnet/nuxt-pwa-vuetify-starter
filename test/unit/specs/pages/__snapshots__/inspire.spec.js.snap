// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inspire renders to a snapshot 1`] = `
<vlayout-stub
  align-center="align-center"
  column="column"
  justify-center="justify-center"
  tag="div"
>
  <vflex-stub
    class="text-xs-center"
    tag="div"
  >
    <div
      class="my-5"
    >
      <nuxt-logo-stub />
       
      <vuetify-logo-stub />
    </div>
     
    <blockquote
      class="blockquote"
    >
      <div>
        “First, solve the problem. Then, write the code.”
      </div>
       
      <footer>
        <small>
          <em>
            —John Johnson
          </em>
        </small>
      </footer>
    </blockquote>
  </vflex-stub>
</vlayout-stub>
`;
